src/styles/global/variables/_color.scss

```diff
@@ -0,0 +1,12 @@
+// Color palette -- All color variables should start with color-
+// ---------------------------------------------------
+
+$color: (
+  '--color-black': #000,
+  '--color-blueprint': #0065f2,
+  '--color-blueprint-bright': #408cf6,
+  '--color-blueprint-soft': #5a9af4,
+  '--color-blueprint-grayish-light': #e6f1fc,
+  '--color-blueprint-pale-very': #f2f8ff,
```

Comment:
At first glance I'm not sure where `--color-blueprint-pale-very` fits in the scale with `--color-blueprint-bright`, `--color-blueprint-soft`, etc. feels somewhat confusing. I'm thinking it may be before `--color-blueprint-soft` I also could feel like it may be between `--color-blueprint-soft` and `--color-blueprint`

That said, what do you think about a numeric scale like [this](https://tailwindcss.com/docs/customizing-colors#color-palette-reference). Granted this may be a little harder if you do not lock down the scale at the start, though I find having a scale like this helps (forces?) you lock it down earlier in the project.